library/Rules/Tld.php

```diff
@@ -121,6 +121,12 @@ class Tld extends AbstractRule
 
     public function validate($input)
     {
-        return in_array(strtolower($input), $this->tldList);
+        // prefere LayerShifter\TLDDatabase if available
```

Comment:
Is that comment really necessary?

Reply:
Might be easier to understand if one is not sure what's going on here